pulp_cli/config.py

```diff
@@ -58,6 +61,9 @@
     ),
     click.option("--username", default=None, help=_("Username on pulp server")),
     click.option("--password", default=None, help=_("Password on pulp server")),
+    click.option("--client_id", default=None, help=_("client_id on Identity Provider")),
+    click.option("--client_secret", default=None, help=_("client_secret on Identity Provider")),
+    click.option("--token_url", default=None, help=_("URL to reach the Identity Provider")),
```

Comment:
The user does not know that. It must be extracted from the www-authenticate header.

Reply:
Although the OAuth2 RFC presents a `WWW-Authenticate` section [here](https://www.rfc-editor.org/rfc/rfc6750#section-3), it's not clear that it should contain the issuer URL where we could ask for a new token, for example. 
Also, I could be wrong here.